tune-threads/utilities.lisp

```diff
@@ -12,15 +12,14 @@
 (defun findmin (list)
```

Comment:
Would it make sense to add an `(assert (not (null list)))` at the top of this function?

Reply:
Yes, I'll make this change.